apps/challenges/github_utils.py

```diff
@@ -0,0 +1,213 @@
+import requests
+import logging
+import base64
+import yaml
+
+from base.utils import deserialize_object
+from .github_sync_config import (
+    challenge_non_file_fields,
+    challenge_file_fields,
+    challenge_phase_non_file_fields,
+    challenge_phase_file_fields,
+)
+from evalai.celery import app
+
+logger = logging.getLogger(__name__)
+
+URLS = {"contents": "/repos/{}/contents/{}", "repos": "/repos/{}"}
+
+
+class GithubInterface:
```

Comment:
@savish28 can we move this class to a separate file called `github_interface.py`

Reply:
Yeah sure :) Done.